### PR TITLE
nerfs directional blocking

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
-blocking has been removed and nerfed on many items
-blocking (and force strings) are now properly communicated when examining an item
-blocking now causes minor hitstun- delays your next click about as long as a melee attack would
-shields now block projectiles again

## Why It's Good For The Game
When i made blocking, my true intent was some counterplay to stunbatons and other melee weapons, and slow down combat, but it was poorly balanced and communicated. I've rectified

## Changelog
:cl:
add: examining an item now tells its blocking ability, blocking arc, and the force string you see on a mouse over
balance: shields can now block projectiles again
balance: blocking has been nerfed on many items
balance: blocking itself has had a minor nerf- blocking an attack delays your next action as if you have hit with a melee attack
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
for the record, it's always been "directional blocking", not "Zeskoblocking", since names are on the name stuff

**Comment by FriendlyContractor** [comment]
Stuff like wrench or crowbar probably shouldnt have blocking. But why nerf more obscure items like mop, rod of asclepicus or synthetic arm blade ? Also i think delay on attacking when you block is horrible and practicaly removes any sort of offensive blocking from the game - person can hit desworder with a pen and they will be completly unable to fight back.

**Comment by BrumalLake** [comment]
fran please come it's weapon tags again with literal force values instead

**Comment by francinum** [comment]
> fran please come it's weapon tags again with literal force values instead

I'm not thrilled, but I also have no argument since it's existing data.

**Comment by zeskorion** [comment]
> fran please come it's weapon tags again with literal force values instead

Force string, not force value. Its what appears when you mouse over an item

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -375,6 +375,7 @@
 	attack_verb = list("sawed", "tore", "cut", "chopped", "diced")
 	hitsound = 'sound/weapons/chainsawhit.ogg'
 	tool_behaviour = TOOL_SAW
+	block_upgrade_walk = 0
 	toolspeed = 1.5 //slower than a real saw
```
Comment (PowerfulBacon) [inline code/game/objects/items/holy_weapons.dm]: Why nerf only the chainsaw sword when every other claymore nullrod has exactly the same stats? (Claymore, darkblade, force weapon, energy sword, etc.)

Comment (zeskorion) [reply code/game/objects/items/holy_weapons.dm]: Ah, i'd meant to nerf the chainsaw arm instead

**Review by francinum — APPROVED** [review]

**Pull request merged**